uniffi_bindgen/src/bindings/python/templates/ErrorTemplate.py

```diff
@@ -0,0 +1,51 @@
+class RustError(ctypes.Structure):
+    _fields_ = [
+        ("code", ctypes.c_int32),
+        ("message", ctypes.c_char_p),
+    ]
+
+    def __str__(self):
+        return "RustError(code={}, message={})".format(self.code, str(self.message, "utf-8")) 
+
+{% for e in ci.iter_error_definitions() %}
+class {{e.name()}}:
+    {%- for value in e.values() %}
+    class {{value}}(Exception):
+        pass
+    {%- endfor %}
```

Comment:
I think this is a good place to start, but having nested class for each error like this is pretty un-idiomatic in python. I'm not sure how to expose our errors in a better way though, so I'm happy to land it like this for now, and we can explore other options later.

Reply:
(Just to be clear: I'm not asking you to change this, I'm just thinking out load about whether there's a better way to do it that we could explore in future)

Reply:
I've created https://github.com/mozilla/uniffi-rs/issues/223 to keeping track on this!